css/jquery.terminal.css

```diff
@@ -684,14 +684,18 @@ terminal .terminal-output > div {
     margin-bottom: 10px;
     position: relative;
 }
+div[data-index] > div {
+    width: -moz-fit-content;
+    width: -webkit-fit-content;
+    width: fit-content;
+}
```

Comment:
It will not work in IE, but I think that this not a big deal. This only need to be documented. that echo newline don't work with IE.